DataAccess/DBContexts.cs

```diff
@@ -33,8 +33,11 @@ public class RetailDbContext : DbContext
         public DbSet<StoreProductTran> StoreProductTrans { get; set; }
         public DbSet<Stocktake> Stocktakes { get; set; }
         public DbSet<Stocktakeline> Stocktakelines { get; set; }
+        public DbSet<PO> Pos { get; set; }
+        public DbSet<POline> Polines { get; set; }
 
-        public RetailDbContext() : base((new SqlConnectionStringBuilder(ConfigurationManager.ConnectionStrings["MyConnectionString"].ConnectionString)).InitialCatalog)
+        public RetailDbContext() : base("Data Source=6800K;Initial Catalog= HODB;Integrated Security=True;Connect Timeout=30;Encrypt=False;TrustServerCertificate=False;ApplicationIntent=ReadWrite;MultiSubnetFailover=False")
+            //(new SqlConnectionStringBuilder(ConfigurationManager.ConnectionStrings["MyConnectionString"].ConnectionString)).InitialCatalog)

```

Comment:
For testing i just threw in my local connection string, you'll probably need to swap it back to the commented line.

Reply:
No problem